### PR TITLE
Partner sentinelone adoption start

### PR DESCRIPTION
**Pull request by anirudh-metron**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/SentinelOne/README.md [hunk 1]
```diff
@@ -11,3 +11,4 @@ The **Sentinel One - Endpoint data collection** playbook collects endpoint infor
 The pack includes the **SentinelOne v2** integration and the **Sentinel One - Endpoint data collection** playbook.
 ## How does this pack work?
 Create an instance of the **SentinelOne v2** integration and start fetching information from the SentinelOne database.
+Note: Support for this Pack will be moved to the Partner on march, 20, 2023.
```
Comment (mchasepan) [inline Packs/SentinelOne/README.md:14]:
```suggestion
Note: Support for this Pack will be moved to the Partner on Feb 1, 2023.
```
I approve a shorter time.

**Comment by kgal-pan** [comment]
@anirudh-metron - Thanks for the contribution. As mentioned in our conversation over Slack and previously in https://github.com/demisto/content/pull/22126 (the reason why it was closed), instead of `SyncIncidents` script, please add [Mirroring](https://xsoar.pan.dev/docs/integrations/mirroring_integration) to the integration.

**Comment by anirudh-metron** [comment]
We are working on the mirroring functionality and update the PR once done.

**Comment by munna-metron** [comment]
HI @kgal-pan, 
Using this same PR for the adoption process will create a lot of conflicts so decided to create a separate PR for the adoption process, here is the [link](https://github.com/demisto/content/pull/23740) for the new PR. And I am going to close this PR.
Thanks!

**Pull request closed without merging**